layouts/index.json

```diff
@@ -0,0 +1,5 @@
+{{- $.Scratch.Add "index" slice -}}
```

Comment:
This needs a different file name to work - I saved it as layouts/search-index/single.html and then was able to get it to do the local search magic. :)

Reply:
Oh. I though it would need to have the .json extension.

Reply:
I can't make it work. What other changes did you apply to the files?